packages/react-devtools-shared/src/utils.js

```diff
@@ -934,6 +937,27 @@ export function formatDataForPreview(
       } catch (error) {
         return 'unserializable';
       }
+    case 'thenable':
+      switch (data.status) {
+        case 'pending':
+          return 'pending Thenable';
+        case 'fulfilled':
+          if (showFormattedValue) {
+            const formatted = formatDataForPreview(data.value, false);
+            return `fulfilled Thenable {${truncateForDisplay(formatted)}}`;
```

Comment:
Not too much of a fan of `fulfilled Thenable {someString}`. Open to suggestions.

Reply:
If we really want to be accurate about naming, then `Thenable` is probably the best option, but I would expect it to be `Promise` in 99.9 % of cases in user land. Both options are okay to me.

Reply:
Yeah, same concern here. I'll just use the name logic we use for `class_instance` unless it's a plain object which is the only case where we fallback to "Thenable".

So you'd actually get "Promise" and when you do [subclassing and pre-seed the special properties](https://bsky.app/profile/sebmarkbage.calyptus.eu/post/3lku7b7xnjk2w), you'd also get the right name